src/pages/GithubReposPage/components/RepoTile/RepoTile.tsx

```diff
@@ -0,0 +1,35 @@
+import "./RepoTile.css";
+import Avatar from "@components/Avatar";
+import { getDay, getMonth } from "@utils/formatTime";
+import { ReactChild, ReactFragment, ReactPortal, SetStateAction, useEffect, useState } from 'react';
+import { RepoItem } from '../../../../store/GithubStore/types';
+import { EventType } from '@testing-library/react';
+import GithubStore from "../../../../store/GithubStore";
+
+const RepoTile = (props: { key: number; tile: any; avatar: string; children: ReactChild; onClick: React.MouseEventHandler<HTMLDivElement> }) => {
```

Comment:
про то, как использовать пропсы, написала ниже

```suggestion
const RepoTile = (props: { key: number; tile: any; avatar: string; children: ReactChild; onClick: (e: React.MouseEvent) => void }) => {
```